multiqc/utils/search_patterns.yaml

```diff
@@ -71,6 +71,9 @@ flexbar:
 gatk/varianteval:
     contents: '#:GATKTable:TiTvVariantEvaluator'
     shared: true
+gatk/base_recalibrator:
+    contents: '#:GATKTable:Arguments:Recalibration argument collection values used in this run'
```

Comment:
I'm not sure what the shortest unique string is, but I am happy to use your suggestion.